src/vm/comutilnative.cpp

```diff
@@ -1922,6 +1922,21 @@ FCIMPL0(int, GCInterface::GetMaxGeneration)
 }
 FCIMPLEND
 
+FCIMPL0(INT64, GCInterface::GetAllocatedBytesForCurrentThread)
+{
+    FCALL_CONTRACT;
+
+    INT64 currentAllocated = 0;
+    Thread *pThread = GetThread();
+    if (pThread)
+    {
+        alloc_context* ac = pThread->GetAllocContext();
+        currentAllocated = ac->alloc_bytes + ac->alloc_bytes_loh - (ac->alloc_limit - ac->alloc_ptr);
```

Comment:
it returns 0. if it's not using the alloc context on the thread, it's using the gen0 alloc context and then we can't separate what got allocated on one thread from another. I will mention issues like this in the documentation for the API.